Apollo/Devices/Multi.cs

```diff
@@ -44,6 +44,21 @@ public class Multi: Device, IMultipleChainParent, ISelectParent {
         public Chain Preprocess;
         public List<Chain> Chains = new List<Chain>();
 
+        List<bool[]> _filters = new List<bool[]>();
+        List<bool[]> Filters {
+            get => _filters;
+            set {
+                int index = 0;
+                foreach (bool[] i in value) {
+                    if (i != null && i.Length == 101) {
+                        _filters[index] = i;
+                        if (Viewer?.SpecificViewer != null) ((KeyFilterViewer)Viewer.SpecificViewer).Set(_filters[index]);
```

Comment:
you're casting to keyfilterviewer instead of multiviewer new set method, this will 300% crash